src/components/layout/footer.module.scss

```diff
@@ -9,16 +9,30 @@
   margin-top: 3rem;
   padding: 2rem 0;
 
+  @media (max-width: $viewport-ms) {
+    .footer-list {
+      flex-direction: column;
+    }
+  }
+
   @media (min-width: $viewport-md) {
+    a:first-child {
```

Comment:
Since all the `a` elements are in `li`, they will always be first children. I think we can just remove this, as there was some wrapping in several browsers I tested:

![image](https://user-images.githubusercontent.com/270536/84092190-de2db900-a9b3-11ea-9201-4e3b61ee2316.png)

Reply:
Thanks, good catch! Fixed in 09d2b649d5b17a78324f5aea